bin/ch/WScriptJsrt.cpp

```diff
@@ -325,11 +334,46 @@ JsErrorCode WScriptJsrt::InitializeModuleInfo(JsValueRef specifier, JsModuleReco
     return JsNoError;
 }
 
-JsErrorCode WScriptJsrt::LoadModuleFromString(LPCSTR fileName, LPCSTR fileContent)
+bool WScriptJsrt::GetFullPath(LPCSTR fileName, __out char* const fullPathNarrow)
+{
+    size_t len = 0;
+    if (fileName)
+    {
+        if (_fullpath(fullPathNarrow, fileName, _MAX_PATH) == nullptr)
+        {
+            return false;
+        }
+        // canonicalize that path name to lower case for the profile storage
+        // REVIEW: This doesn't work for UTF8...
+        len = strlen(fullPathNarrow);
+        for (size_t i = 0; i < len; i++)
+        {
+            fullPathNarrow[i] = (char)tolower(fullPathNarrow[i]);
+        }
+    }
+    else
+    {
+        // No fileName provided (WScript.LoadScript()), use dummy "script.js"
+        strcpy_s(fullPathNarrow, 10, "script.js");
```

Comment:
Shouldn't this be expanded to full path too?
It is probably better to default script.js outside of GetFullPath.